dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -0,0 +1,45 @@
+class EvaluateRubricJob < ApplicationJob
+  queue_as :default
+
+  rescue_from(StandardError) do |exception|
+    if rack_env?(:development)
+      puts "EvaluateRubricJob Error: #{exception.full_message}"
+    end
+    raise
+  end
+
+  # 2D Map from unit name and level name, to the name of the lesson files in S3
+  # which will be used for AI evaluation.
+  # TODO: This is a temporary solution. After the pilot, we should at least make
+  # the S3 pointer editable on levelbuilder, and eventually make all of the data
+  # it points to editable there too.
+  UNIT_AND_LEVEL_TO_LESSON_S3_NAME = {
+    'csd3-2023' => {
+      'CSD U3 Interactive Card Final_2023' => 'CSD-2022-U3-L17',
+      'CSD U3 Sprites scene challenge_2023' => 'New-U3-2022-L10',
+      'CSD web project animated review_2023' => 'New-U3-2022-L13',
+      'CSD games sidescroll review_2023' => 'New-U3-2022-L20'
+    }
+  }
+
+  def perform(user_id:, script_level_id:)
+    user = User.find(user_id)
+    script_level = ScriptLevel.find(script_level_id)
+    lesson_s3_name = EvaluateRubricJob.get_lesson_s3_name(script_level)
+    puts "Evaluating rubric for user #{user.id} on script level #{script_level.id} with lesson_s3_name: #{lesson_s3_name.inspect}"
```

Comment:
Do we want this puts on the production job?

Reply:
good call. based on the notes at the end of https://github.com/code-dot-org/code-dot-org/pull/53976, I feel reasonably confident that this line can be removed and we'll still have the relevant job params logged to somwhere we can find them, so I'll go ahead and remove this

Reply:
I realize now I included these statements to avoid the UselessAssignment linter warnings. I've changed this logging to be in development only, where IIRC it is still somewhat useful

Reply:
ok, apologies, I'm realizing I'm taking a shortcut to avoid merge conflicts because i've stacked another PR on top of this one. I'll go ahead and completely remove this in the following PR: baeb880938d9239fcbd6c3b740d591cc4e5ac75d